tasks/sast-java-sec-check.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sast-java-sec-check
+spec:
+  workspaces:
+    - name: workspace
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  params:
+    - name: PATH_CONTEXT
+      description: The path to your source code
+      type: string
+      default: .
+    - name: OUTPUT_FORMAT
+      description: the format of findsecbugs output
+      type: string
+      default: "sarif"
+    - name: OUTPUT_ONLY_ANALYZE
+      description: |
+        only analyze given classes and packages;
+        end with .* to indicate classes in a package, .- to indicate a package prefix
+      type: string
+      default: ""
+    - name: OPTIONAL_ARGS
+      description: optional parameters to run findsecbugs
+      type: string
+      default: ""
+  steps:
+    - name: java-sec-check
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      image: quay.io/redhat-appstudio/hacbs-test:feature-sast
```

Comment:
this is temporary?

Reply:
We are using that tag/branch for all of the SAST feature tasks while they are being developed. Once we have all the changes in, we will see how well the full container works (mainly because of its size and number of layers). If that looks good, we'll raise a PR to the main branch and cut a release.

Reply:
ok, thanks